src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs

```diff
@@ -39,81 +39,81 @@ namespace Tizen.NUI.Scene3D
     /// orientation.TransformType = MotionTransformIndex.TransformTypes.Orientation;
     /// </code>
     /// </example>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 11 </since_tizen>
     public class MotionTransformIndex : MotionIndex
     {
         /// <summary>
         /// The list of component types what this MotionIndex can control.
```

Comment:
component or properties? same in line 158

Reply:
I make it as transform property types,